rules/falco_rules.yaml

```diff
@@ -2494,9 +2494,10 @@
 - rule: Unexpected K8s NodePort Connection
   desc: Detect attempts to use K8s NodePorts from a container
   condition: (inbound_outbound) and fd.sport >= 30000 and fd.sport <= 32767 and container and not nodeport_containers
+  enabled: false
```

Comment:
condition resolves to `and container and not container`, we missed that this can never evaluate to true if not tuned.